### ISSUE TITLE
Change Rocket.Chat data path

### ISSUE THREAD

**Issue by thumbapps** [RocketChat/Rocket.Chat.Electron#383]
## My Setup

 - __My OS name and version__: Windows 10
 - __My current app version__: 2.5.0

- [x] I have tested with the latest application version
- [x] I can simulate the issue easily

#### Current Behavior

Right now Rocket.Chat for Desktop/Win can only store its data in %APPDATA%

#### Expected Behavior

It would be nice if user can use some command line option or environment variable or any other means to change Rocket.Chat's data path, eg. to a folder in its program folder.

In short, a portable mode :)

Thanks!

**Comment by thehutman** [comment]
Windows client 2.5.0 -- can't download any file attachments either.  Works fine in browser.  Windows client doesn't do anything when you click a file to download.

**Comment by klou** [comment]
Conversely, a portable executable (that uses %APPDATA%) would be nice as well.
